library/core/src/net/ip_addr.rs

```diff
@@ -1773,6 +1775,8 @@ impl Ipv6Addr {
             && !self.is_unspecified()
             && !self.is_documentation()
             && !self.is_benchmarking()
+            // Segment Routing (SRv6) SIDs (`5f00::/16`)
+            && !matches!(self.segments(), [0x5f00, ..])
```

Comment:
🐃 I haven't changed this to something potentially more comprehensive like `self.is_unicast() && !self.global()` since that would change behaviour for some other ranges, but it may be a good target for another PR.

Reply:
See #85604 -- I think given those open questions, we should not change this at all right now.

(The last `is_global` update in #119006 also avoided this.)

Reply:
I will have a look and re-request review when this is ready.

@rustbot author